tests/functional/start.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+xunitfile=/mnt/test/casperesults.xml
+
+echo The base domain is $BASE_DOMAIN
+
+echo Starting S3 Monitor
+php /mnt/test/functional/service/S3Monitor.php &
+
+# echo Removing ${xunitfile}
+# rm -f ${xunitfile}
+
+echo Starting CasperJS Tests
+/usr/local/bin/casperjs test /mnt/test/functional/$1 --ignore-ssl-errors=true --ssl-protocol=any --includes=/mnt/test/functional/config/Bootstrap.js  --xunit=${xunitfile}
+
+RETVAL=$?
+echo printing $RETVAL
+
+if [ $RETVAL -eq 0 ]; then
+    echo OK
+else
+    echo FAIL
+fi
+
+echo Changing permissions on ${xunitfile}
+# We are running as root so make xunit results deletable
```

Comment:
These comment lines aren't adding anything more that the echo isn't already telling us